client/http/client.go

```diff
@@ -399,10 +412,10 @@ func (c *client) GetHistoryHotRegions(ctx context.Context, req *HistoryHotRegion
 
 // GetRegionStatusByKeyRange gets the region status by key range.
 // The keys in the key range should be encoded in the UTF-8 bytes format.
-func (c *client) GetRegionStatusByKeyRange(ctx context.Context, keyRange *KeyRange) (*RegionStats, error) {
+func (c *client) GetRegionStatusByKeyRange(ctx context.Context, keyRange *KeyRange, onlyCount bool) (*RegionStats, error) {
```

Comment:
I noticed some missing introductions in `func (h *statsHandler) GetRegionStatus(w http.ResponseWriter, r *http.Request) {`, such as explaining the role of `count`, can you update the comment?